lib/config.ts

```diff
@@ -180,6 +180,26 @@ export function get(name: string): any
                     arg: 'websocket-ws-port'
                 },
             }
+        },
+        'longPoll': {
```

Comment:
For this line and the remaining part of this section of the config files, let's lower-case everything and use `-` where appropriate.  I think `longpoll` is acceptable.